pkg/api/configsync/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -5,7 +5,7 @@
 package v1alpha1
 
 import (
-	"k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	v1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
```

Comment:
This doesn't seem related to this change. Is this just a change with Go 1.22? I haven't seen this locally.

Reply:
Reverted